### PR TITLE
EvenOdd Python

### LINKED ISSUE #1467: number of even and odd elements in an array
I would like to do that

### PR DESCRIPTION
**Pull request by raksha009**
Fixes #1467 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Addition of new code
- 

#### Languages Used:
- Python
- 

#### Files Added:
- EvenODD.py
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please remove the files which is not relevant to the issue and let me know

**Comment by raksha009** [comment]
@asha15  plz add EvenODD python file from raksha009 account (i.e the second commit). It was my first PR, hence it created confusion. Sorry for inconvenience.

**Comment by asha15** [comment]
from your side you need to remove this file. 
please reach me on slack

File: 1-D_Array/Python/EvenOdd.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Python program to count even and odd numbers in an array
+
+# To count number of even elements
+def  no_of_even_elem(arr):
```
Comment (asha15) [inline 1-D_Array/Python/EvenOdd.py]: have kept two spaces after def keyword? if yes remove a space

File: 1-D_Array/Python/EvenOdd.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    return even_num
+            
+
+#--- main--- function
+
+# initialising variables
+arrlength = 0
+num  = 0
+
+#input array from user
```
Comment (asha15) [inline 1-D_Array/Python/EvenOdd.py]: remove this line

File: 1-D_Array/Python/EvenOdd.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+lst = []    # creating an empty list
```
Comment (asha15) [inline 1-D_Array/Python/EvenOdd.py]: take this commend to the previous line

File: 1-D_Array/Python/EvenOdd.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+# initialising variables
+arrlength = 0
+num  = 0
+
+#input array from user
+
+lst = []    # creating an empty list 
+  
+n = int(input("Enter number of elements : ")) # number of elements of list
```
Comment (asha15) [inline 1-D_Array/Python/EvenOdd.py]: take this commend to the previous line

Comment (raksha009) [reply 1-D_Array/Python/EvenOdd.py]: @asha15  I modified the commit and made the required changes.

**Comment by maahi07m** [comment]
Either you can use Comprehension way to print Even,Odd

> num_list = input()
[print(i) for i in num_list if i%2==0]
[print(i) for i in num_list if i%2!=0]

**Comment by raksha009** [comment]
> Either you can use Comprehension way to print Even,Odd
> 
> > num_list = input()
> > [print(i) for i in num_list if i%2==0]
> > [print(i) for i in num_list if i%2!=0]

Yes, but the way i did is also correct na. Because the numbers will be either odd or even.

File: 1-D_Array/Python/EvenOdd.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    # Initialization
+    even_num = 0
+
+    # Finding even numbers in array
+    for i in range (0,len(arr)):
+        if arr[i] %2 == 0:
+            even_num +=1
+
+    return even_num
+
```
Comment (asha15) [inline 1-D_Array/Python/EvenOdd.py]: remove this line

File: 1-D_Array/Python/EvenOdd.py [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+ # number of elements of list 
+n = int(input("Enter number of elements : "))
+  
+for i in range(0, n): 
+    elem = int(input()) 
+    lst.append(elem) # adding the element 
+
+num = no_of_even_elem(lst)
+print "Number of odd elements in array", len(lst)- num
```
Comment (asha15) [inline 1-D_Array/Python/EvenOdd.py]: keep space before minus

**Comment by asha15** [comment]
please squash the commits before pushing it

File: 1-D_Array/Python/EvenOdd.py (hunk 6 above)
Comment (raksha009) [reply 1-D_Array/Python/EvenOdd.py]: @asha, I've made all the suggested changes. pl review

**Comment by asha15** [comment]
Good job :)

**Pull request merged**